CryptoPrices/Sources/Data/Sources/NetworkExtension/CoinRoute.swift

```diff
@@ -23,15 +23,17 @@ extension CoinRoute: Route {
 
     public var path: String {
         switch self {
-        case .myCoins: return "/coins/market"
-        case .trendingCoins: return "/coins/markets"
+        case .myCoins, .trendingCoins: return "/coins/markets"
```

Comment:
👍so the `myCoins` API also uses the path that has `markets` with `s` 🤣

Reply:
Yeah 🤣, and it's actually my bad (typo) when implementing the previous backend task 😅